index.html

```diff
@@ -143,6 +143,31 @@ <h2 class="text-center text-bold text-lucida text-subheading mb-5">Testimonials<
             </div>
         </div>
     </section>
+    <footer id="footer" class="container">
+        <div class="d-flex flex-wrap pt-5 pb-5">
```

Comment:
Looks like it has 4 spaces indentation here but the code below uses 2-spaces indentation.

Please use consistent indentation.